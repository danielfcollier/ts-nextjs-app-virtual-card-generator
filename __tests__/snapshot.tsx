import { render } from '@testing-library/react';
import Generate from '@/pages/generate';

it('renders homepage unchanged', () => {
  const { container } = render(<Generate />);
  expect(container).toMatchSnapshot();
});
