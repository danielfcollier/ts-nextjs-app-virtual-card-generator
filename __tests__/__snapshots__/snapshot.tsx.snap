// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      QR Code Image Generator
    </h1>
    <form>
      <label>
        Name
        <input
          id="name"
          name="name"
          type="text"
          value="Daniel"
        />
        <br />
      </label>
      <label>
        LinkedIn URL
        <input
          class="cards"
          id="linkedin"
          name="linkedin"
          type="text"
          value="https://linkedin.com/in/danielfcollier"
        />
        <br />
      </label>
      <label>
        GitHub URL
        <input
          class="cards"
          id="github"
          name="github"
          type="text"
          value="https://github.com/danielfcollier"
        />
        <br />
      </label>
      <input
        type="submit"
        value="Generate Image"
      />
    </form>
  </div>
</div>
`;
